src/main/java/com/VetApp/PurrgentCare/models/Account.java

```diff
@@ -29,13 +30,29 @@ public class Account {
     @OneToMany(cascade = CascadeType.REMOVE,
             mappedBy = "account", orphanRemoval = true)
     private List<Person> accountHolders;
-
     @JsonManagedReference
     @Setter
     @OneToMany(cascade = CascadeType.REMOVE,
             mappedBy = "account", orphanRemoval = true)
     private List<Pet> pets;
 
+    public void addAccountHolder(Person accountHolder) {
+        accountHolder.setAccount(this);
+        //List would not allow addition of accountHolders because it is immutable
+        //Had to convert to ArrayList to make it mutable and then set object List
+        List<Person> accountHolders = new ArrayList<>(getAccountHolders());
+        accountHolders.add(accountHolder);
+        setAccountHolders(accountHolders);
+    }
+
+    public void addAccountHolders(List<Person> accountHolders) {
+        for (Person accountHolder : accountHolders) {
+            if (!getAccountHolders().contains(accountHolder)) {
+                addAccountHolder(accountHolder);
+            }
+        }
+    }
+
```

Comment:
Abstracted addAccountHolders helper method to Account model.

Had some trouble with accountHolders being an immutable list that was causing the associatePeople_whenAccountExists_returnAssociatedPeople test to fail with an Unsupported Operation Exception error.
Fixed that by converting to ArrayList.

Is there a reason we want the accountHolders to be immutable rather than mutable?